src/vars.c

```diff
@@ -2008,13 +2008,16 @@ static Obj FuncParentLVars(Obj self, Obj lvars)
 
 static Obj FuncContentsLVars(Obj self, Obj lvars)
 {
+  if (!IS_LVARS_OR_HVARS(lvars)) {
+      RequireArgument("ParentLVars", lvars, "must be an lvars");
+  }
   Obj contents = NEW_PREC(0);
   Obj func = FUNC_LVARS(lvars);
   Obj nams = NAMS_FUNC(func);
   UInt len = (SIZE_BAG(lvars) - 2*sizeof(Obj) - sizeof(UInt))/sizeof(Obj);
   Obj values = NEW_PLIST_IMM(T_PLIST, len);
   if (lvars == STATE(BottomLVars))
-    return False;
+    return Fail;
```

Comment:
I assume then the test (what travis tests) should be changed accordingly.

Reply:
Updated